src/Xpress.jl

```diff
@@ -43,7 +45,7 @@ module Xpress
     )
 
     function initialize()
-        Libdl.dlopen(joinpath(dlpath, libxprs))
+        lib = Libdl.dlopen(libxprs)
```

Comment:
we don't need `lib = ` right?

Reply:
Resolved in https://github.com/jump-dev/Xpress.jl/pull/110/commits/c0db24ed62b9897cea8bbf742da2c17b1a581a34.